kubecost/templates/aggregator/aggregator-servicemonitor.yaml

```diff
@@ -28,4 +28,40 @@ spec:
   selector:
     matchLabels:
     {{- include "kubecost.aggregator.commonLabels" . | nindent 6 }}
+    {{- if .Values.aggregator.service.labels }}
+    {{- toYaml .Values.aggregator.service.labels | nindent 6 }}
+    {{- end }}
+---
+apiVersion: monitoring.coreos.com/v1
```

Comment:
we may change this before GA. but I don't see a reason to not enable both when enabling one for now.